crates/driver/src/tests/cases/mod.rs

```diff
@@ -21,28 +23,51 @@ const SOLVER_NAME: &str = "test1";
 
 /// The default surplus factor. Set to a high value to ensure a positive score
 /// by default. Use a surplus factor of 1 if you want to test negative scores.
-pub const DEFAULT_SURPLUS_FACTOR: u64 = 10000000000u64;
+pub const DEFAULT_SURPLUS_FACTOR: f64 = 1e-8;
 
-pub const DEFAULT_POOL_AMOUNT_A: u128 = 100000000000000000000000u128;
-pub const DEFAULT_POOL_AMOUNT_B: u128 = 6000000000000000000000u128;
-pub const DEFAULT_POOL_AMOUNT_C: u128 = 100000000000000000000000u128;
-pub const DEFAULT_POOL_AMOUNT_D: u128 = 6000000000000000000000u128;
+pub const DEFAULT_POOL_AMOUNT_A: u64 = 100000;
+pub const DEFAULT_POOL_AMOUNT_B: u64 = 6000;
+pub const DEFAULT_POOL_AMOUNT_C: u64 = 100000;
+pub const DEFAULT_POOL_AMOUNT_D: u64 = 6000;
 
 /// The order amount for orders selling token "A" for "B".
-pub const AB_ORDER_AMOUNT: u128 = 50000000000000000000u128;
+pub const AB_ORDER_AMOUNT: u64 = 50;
 
 /// The order amount for orders selling token "C" for "D".
-pub const CD_ORDER_AMOUNT: u128 = 40000000000000000000u128;
+pub const CD_ORDER_AMOUNT: u64 = 40;
 
-pub const ETH_ORDER_AMOUNT: u128 = 40000000000000000000u128;
+pub const ETH_ORDER_AMOUNT: u64 = 40;
 
 /// With the default amounts defined above, this is the expected score range for
 /// both buy and sell orders.
-pub const DEFAULT_SCORE_MIN: u128 = 2000000000000000000u128;
-pub const DEFAULT_SCORE_MAX: u128 = 500000000000000000000000000000u128;
+pub const DEFAULT_SCORE_MIN: u64 = 2;
+pub const DEFAULT_SCORE_MAX: u64 = 500000000000;
 
 /// The default solver fee for limit orders.
-pub const DEFAULT_SOLVER_FEE: u128 = 100u128;
+pub const DEFAULT_SOLVER_FEE: f64 = 1e-16;
 
 /// The default maximum value to be payout out to solver per solution
-pub const DEFAULT_SCORE_CAP: u128 = 10000000000000000u128;
+pub const DEFAULT_SCORE_CAP: f64 = 1e-2;
+
+pub trait IntoWei {
+    fn into_wei(self) -> eth::U256;
+}
```

Comment:
Please document this trait. 

I don't find it completely intuitive because it's not obvious that `3.into_wei()` means 3 ETH into wei (it could also mean 3 Wei). 
Maybe `3.ether()` would be more intuitive (or having a `struct Ether(U256)` and then doing `Ether::from(3).into_wei()`)?

Reply:
Added a wrapper